### DISCUSSION TITLE
Organizations list should be sorted

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44013] [Mobile] [answered]
### Select Topic Area

Bug

### Body

* https://github.com/settings/organizations on desktop shows an alphabetically sorted list of organizations
* the equivalent view in GitHub Mobile (Profile>Organizations) is a chronological list based on join order

## Expected results

The order should match the desktop view

**Comment by HenSquared** [comment, marked as answer]
Thanks for the feedback. I've opened up an Issue with the API team to see if we can match sort order to dotcom
